static/js/containers/CourseSearchPage.js

```diff
@@ -329,6 +294,5 @@ export default R.compose(
   connect(
     mapStateToProps,
     mapDispatchToProps
-  ),
-  withLoading(CourseSearchLoading)
+  )
```

Comment:
since you removed the `withLoading` component here you can also get rid of the `R.compose` call